### PR TITLE
Add gptfdisk package 1.0.8

### PR DESCRIPTION
**Pull request by lbmeng**
This adds a new package for mingw64: gptfdisk.

It includes text-mode partitioning tools that work on GUID Partition
Table (GPT) disks.

The following tools are included:

  - gdisk.exe
  - sgdisk.exe
  - fixparts.exe

Signed-off-by: Bin Meng <bmeng.cn@gmail.com>

### CODE REVIEW

File: mingw-w64-gptfdisk/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  make -f Makefile.mingw64
+}
+
+package() {
+  cd "$srcdir/${_realname}-$pkgver"
+  install -d "$pkgdir"/usr/{bin,share/{doc/gdisk,man/man8}}
+  install -t "$pkgdir/usr/bin" {,s}gdisk.exe fixparts.exe
+  install -m644 -t "$pkgdir/usr/share/man/man8" {{,s}gdisk,fixparts}.8
+  install -m644 -t "$pkgdir/usr/share/doc/gdisk" README NEWS
+}
```
Comment (lazka) [inline mingw-w64-gptfdisk/PKGBUILD:52]: You need to install into `${pkgdir}${MINGW_PREFIX}/`{bin,share}

Comment (lbmeng) [reply mingw-w64-gptfdisk/PKGBUILD]: Thanks, addressed and pushed a newer version.

File: mingw-w64-gptfdisk/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  cd "$srcdir/${_realname}-$pkgver"
+  make -f Makefile.mingw64
+}
+
+package() {
+  cd "$srcdir/${_realname}-$pkgver"
+  install -d "$pkgdir${MINGW_PREFIX}"/usr/{bin,share/{doc/gdisk,man/man8}}
+  install -t "$pkgdir${MINGW_PREFIX}/usr/bin" {,s}gdisk.exe fixparts.exe
+  install -m644 -t "$pkgdir${MINGW_PREFIX}/usr/share/man/man8" {{,s}gdisk,fixparts}.8
+  install -m644 -t "$pkgdir${MINGW_PREFIX}/usr/share/doc/gdisk" README NEWS
```
Comment (lazka) [inline mingw-w64-gptfdisk/PKGBUILD:51]:
```suggestion
  install -d "$pkgdir${MINGW_PREFIX}"/{bin,share/{doc/gdisk,man/man8}}
  install -t "$pkgdir${MINGW_PREFIX}/bin" {,s}gdisk.exe fixparts.exe
  install -m644 -t "$pkgdir${MINGW_PREFIX}/share/man/man8" {{,s}gdisk,fixparts}.8
  install -m644 -t "$pkgdir${MINGW_PREFIX}/share/doc/gdisk" README NEWS
```

Comment (lbmeng) [reply mingw-w64-gptfdisk/PKGBUILD]: Fixed

**Comment by lazka** [comment]
thanks!

**Pull request merged**